apps/examples/wifi_manager/wm_test/Makefile

```diff
@@ -40,7 +40,8 @@ wm_test_daemon.c\
 wm_test_network.c\
 wm_test_event_tc.c\
 wm_test_connect_stable.c\
-wm_test_dns.c
+wm_test_dns.c\
```

Comment:
Could you please modify the following line in the commit description,
"It will check whether TP1x board is connected to access point having best RSSI value" -> It will check whether `the` TP1x board is connected to `the` access point having `the` best RSSI value